app/controllers/admin/admin_users_controller.rb

```diff
@@ -69,27 +69,33 @@ def update
   def update_next_of_kin
     @user = authorize User.find_by!(login: params[:user_login])
     fnok = @user.fannish_next_of_kin
+    previous_fnok_user_id = fnok&.kin&.id
     kin = User.find_by(login: params[:next_of_kin_name])
     kin_email = params[:next_of_kin_email]
 
     if kin.blank? && kin_email.blank?
-      if fnok.present?
+      if fnok.blank?
+        flash[:notice] = ts("No change to Fannish next of kin.")
```

Comment:
Just noting "Fannish" should be lowercase here and on line 90. 

Taking another shot at this this weekend is fine!